pkgs/development/tools/analysis/cargo-tarpaulin/default.nix

```diff
@@ -26,6 +26,6 @@ rustPlatform.buildRustPackage rec {
     homepage = "https://github.com/xd009642/tarpaulin";
     license = with licenses; [ mit /* or */ asl20 ];
     maintainers = with maintainers; [ hugoreeves ];
-    platforms = lib.platforms.x86_64;
+    platforms = lib.platforms.unix;
```

Comment:
`buildRustPackage` already does that
```suggestion
```
https://github.com/NixOS/nixpkgs/blob/d16c79b22de3e5de4fde3342dae29876208ab766/pkgs/build-support/rust/build-rust-package/default.nix#L159